tests/dogtag/pytest-ansible/pytest/performance_test/test_cert_enrollment.py

```diff
@@ -74,7 +78,14 @@ def __init__(self, sn_uid):
         for enrollment_result in enrollment_results:
             request_data = enrollment_result.request
             cert_data = enrollment_result.cert
-            print('Request ID: ' + request_data.request_id)
+            log.info("Request ID: '{}'".format(request_data.request_id))
+
+            # Revoke a certificate after every 100 certificate
+            if (int(request_data.request_id) % 100) == 0:
+                revoke_data=cert_client.revoke_cert(cert_data.serial_number, revocation_reason='Key_Compromise',
+                                        invalidity_date=None, comments="Revoke certificate for testing", nonce=None)
+                log.info("Revoke ID: '{}'".format(revoke_data.request_id))
```

Comment:
Let's do the revocation test in a separate script (e.g. test_cert_revocation.py) so we can measure the performance of enrollment and revocation operations separately. We can probably run it after test_cert_enrollment.py so the database is already populated.

Reply:
Added revocation test in separate script "_test_cert_revocation.py_".